test/README.md

```diff
@@ -21,6 +21,15 @@ To run all the tests
 KUBECONFIG="/home/user/.kube/config" ENABLE_TEST_CASE=reboot PTP_TEST_MODE=Discovery make functests
 ```
 
+To run all the tests with a container:
```

Comment:
I would rather give  them plain  docker command so they don't have to checkout the project 
docker  run -e PTP_TEST_MODE=$TEST_MODE -e ENABLE_TEST_CASE=$ENABLE_TC -v $KUBECONFIG:/tmp/config:Z -v $OUTPUT:/output:Z  $IMAGE

Reply:
fixed